src/item_factory.cpp

```diff
@@ -97,6 +97,9 @@ static bool assign_coverage_from_json( const JsonObject &jo, const std::string &
                                        body_part_set &parts, bool &sided )
 {
     auto parse = [&parts, &sided]( const std::string & val ) {
+        if( is_legacy_bodypart_id( val ) ) {
```

Comment:
So if the json use a legacy id then the item covers no bodypart? Why not just fix jsons instead? Or convert legacy id to new ids?